assets/src/application/ui/layout/dropdownMenu/DropdownMenuItem.tsx

```diff
@@ -4,10 +4,11 @@ import classNames from 'classnames';
 import { MenuItem } from '@infraUI/layout/menu';
 import { DropdownMenuItemProps } from './types';
 
-const DropdownMenuItem: React.FC<DropdownMenuItemProps> = ({ icon: Icon, title, className, ...rest }) => {
-	const newClassName = classNames('components-dropdown-menu__menu-item', className);
```

Comment:
`newClassName` name does not reflect what is happening here;
if this is new, then we should have old one somewhere?